src/cloudsc_gpu/dwarf_cloudsc_gpu.F90

```diff
@@ -42,6 +42,8 @@ PROGRAM DWARF_CLOUDSC
 USE CLOUDSC_DRIVER_GPU_SCC_FIELD_MOD, ONLY: CLOUDSC_DRIVER_GPU_SCC_FIELD
 #endif
 
+USE OMP_LIB
```

Comment:
Guard with `#ifdef _OPENMP`, and in l.75 below

Reply:
resolved in next commit